### ISSUE TITLE
Asterisk in front of properties (for IE7 + 6) result in ruby error.

### ISSUE THREAD

**Issue by michsch** [brigade/scss-lint#85]
If i use an asterisk in front of a property, scss-lint gives the following error as expected:

``` bash
_typography.scss:122 [W] Unknown property *list-style
```

In my configuration file i added this special IE property:

``` yaml
  PropertySpelling:
    enabled: true
    extra_properties: [ *list-style ]
```

Now i got this ruby error:

``` bash
/Users/ms/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/psych/visitors/to_ruby.rb:248:in `block in visit_Psych_Nodes_Alias': Unknown alias: list-style (Psych::BadAlias)
```
